lib/questions.rb

```diff
@@ -1,61 +1,78 @@
 # keep only the elements that start with an a
 def select_elements_starting_with_a(array)
+  array.keep_if {|arr| arr.start_with?'a'}
```

Comment:
Put space between the method name and the first argument.<br>Space between { and | missing.<br>Space missing inside }.